chitterModel.js

```diff
@@ -0,0 +1,33 @@
+class ChitterModel {
```

Comment:
Good. I think the model does what it is suppose to do and names are clear enough to be understood from someone who didn't write the code